app/assets/javascripts/foreman_wreckingball/modal.js

```diff
@@ -0,0 +1,43 @@
+$(function() {
+  $('#confirmation-modal .secondary').click(function(){
+    $('#confirmation-modal').modal('hide');
+  });
+});
+
+function submit_modal_form() {
+  $('#confirmation-modal form').submit();
+  $('#confirmation-modal').modal('hide');
+}
+
+function show_modal(element, url) {
+  if (!url) {
+    url = $(element).attr('href');    
+  }
+  var title = $(element).attr('data-title');
+  if (title) {
+    $('#confirmation-modal .modal-title').text(title);
+  }
+  $('#confirmation-modal .modal-body')
+    .empty()
+    .append('<div class="modal-spinner spinner spinner-lg"></div>');
+  $('#confirmation-modal').modal();
+  $('#confirmation-modal .modal-body').load(url + ' #content',
+      function(response, status, xhr) {
+        $('#confirmation-modal .form-actions').remove();
+        $('#confirmation-modal .col-md-6')
```

Comment:
What is this used for?

Reply:
`trigger_selector` generates a trigger view that's designed to be used in a full-page view, so in the modal it becomes far too narrow.

Going to probably throw this away as I redo the selector generation.